packages/@okta/vuepress-site/docs/guides/submit-app/submission2-specific/openidconnect/submit.md

```diff
@@ -22,11 +22,11 @@
 
 * **Enter your Initiate Login URI** &mdash; Provide the complete login URI for your app integration, including the variable names if you're using a per tenant design.
 
-* **Optional: Do you require a Post Logout Redirect URI?** &mdash; Include a Post Logout Redirect URI if you have a location where you want to send your end user to after they log out of your application.
+* **Optional: Do you require a Post Logout Redirect URI?** &mdash; Include a sign-out Redirect URI if you have a location where you want to send your end user to after they log out of your application.
 
 * **Does your Post Logout URI vary per tenant?** &mdash; If **Yes**, an additional field appears to assist you in setting up a per tenant configuration. The instructions for adding and using the custom variables are identical to the per tenant instructions for the Redirect URI.
 
-* **Enter your Post Logout Redirect URI** &mdash; Provide the complete logout URI for your app integration, including the variable names if you're using a per tenant design.
+* **Enter your Post Logout Redirect URI** &mdash; Provide the complete sign-out redirect URI for your app integration, including the variable names if you're using a per tenant design.
 
 * **To configure OIDC, can your customers do it by themselves from your app's UI, or do they need to contact your support team?** &mdash; If a customer needs support to configure your app integration, you need to include support contact information in your configuration guide. We recommend that you build a UI that enables self-service configuration to reduce the setup time for your customers.
```

Comment:
General comment:
The style guide says "Don’t capitalize the first word after an em dash unless the word is a proper noun."

Reply:
Shoot - I thought it was the other way around. Will fix here and everywhere else in the library.